x/did/handler.go

```diff
@@ -27,14 +27,38 @@ func handleMsgAddDidDoc(ctx sdk.Context, k keeper.Keeper, msg types.MsgAddDid) s
 	if err != nil {
 		return err.Result()
 	}
-	return sdk.Result{}
+
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeAddDidDoc,
+			sdk.NewAttribute(types.AttributeKeyDid, msg.Did),
+			sdk.NewAttribute(types.AttributeKeyPubKey, msg.PubKey),
+		),
+		sdk.NewEvent(
+			sdk.EventTypeMessage,
+			sdk.NewAttribute(sdk.AttributeKeyModule, types.AttributeValueCategory),
+		),
+	})
+	return sdk.Result{Events: ctx.EventManager().Events()}
 }
 
 func handleMsgAddCredential(ctx sdk.Context, k keeper.Keeper, msg types.MsgAddCredential) sdk.Result {
 	err := k.AddCredentials(ctx, msg.DidCredential.Claim.Id, msg.DidCredential)
 	if err != nil {
 		return err.Result()
 	}
-
-	return sdk.Result{}
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeAddCredential,
+			sdk.NewAttribute(types.AttributeKeyCredType, msg.DidCredential.CredType[0]),
+			sdk.NewAttribute(types.AttributeKeyIssuer, msg.DidCredential.Issuer),
+			sdk.NewAttribute(types.AttributeKeyIssued, msg.DidCredential.Issued),
+			sdk.NewAttribute(types.AttributeKeyClaimID, msg.DidCredential.Claim.Id),
```

Comment:
Here we want to output whether the claim is KYC validated `msg.DidCredential.Claim.KYCValidated` not just the claim ID. Please add an attribute for it `AttributeKeyClaimKYCValidated`.

Reply:
added the AttributeKeyClaimKYCValidated